### PR TITLE
Imports pull requests from github and add's them to firebase

### LINKED ISSUE #382: Import PRs from GitHub
We currently do not fully support PRs created with GitHub. It is possible to assume default values for certain things, like PR status, when a PR was not created with our tool. But to make such externally created PRs well reviewable with our tool, things like line-comments, reviewers, approvals etc all have to be imported in some way from GitHub.

### PR DESCRIPTION
**Pull request by eanker**
If a pull request is made via github no data is stored in firebase.
This means that certain features won't work, because the path in firebase does not exists. Therefore (pr specific) default data is stored in firebase, when there is no data stored.

Together with preview-code/backend#16
fixes #382

---

Review this pull request [on Preview Code](https://preview-code.com/projects/preview-code/frontend/pulls/387/overview).

### CODE REVIEW

File: src/projects/diff/hunk-sorter.html [hunk 1]
```diff
@@ -159,6 +170,11 @@
         if (unsortedGroup.diff.length > 0) {
           allGroups.push(unsortedGroup);
         }
+        if (fbOrdering.groups === undefined && this.backendPath && defaultGroup.diff.length > 0) {
+          this.$.checkIfPathIsThere.body = defaultGroup;
```
Comment (txsmith) [inline src/projects/diff/hunk-sorter.html:173]: Maybe we should alter the name of the default group to show the user that the changes were directly imported from GitHub

**Review by txsmith — CHANGES_REQUESTED** [review]
I still need to actually test this, but the code looks good. Just one comment on the default group name

File: src/projects/diff/hunk-sorter.html (hunk 1 above)
Comment (eanker) [reply src/projects/diff/hunk-sorter.html:173]: I like this. Shall make the change.

**Comment by eanker** [comment]
I reverted the rebase for now, as otherwise the backend doesn't work.
Will rebase when the other pr's are merged.

File: src/projects/diff/hunk-sorter.html (hunk 1 above)
Comment (eanker) [reply src/projects/diff/hunk-sorter.html:173]: Should we just change 'Unsorted Changes' to 'Imported from Github' or do we really want to do it only on import?

Comment (txsmith) [reply src/projects/diff/hunk-sorter.html:173]: Unsorted changes always come from GitHub so I think we should just change the `Unsorted changes` string

**Comment by eanker** [comment]
This pr should work now.

There was a problem with posting and then showing the first group comment, but I think I have fixed it.
It was sometimes hard to reproduce (no idea why it sometimes decided not to show), so look carefully at that.

**Review by txsmith — APPROVED** [review]

**Comment by txsmith** [comment]
Confirmed working!

**Pull request merged**